fe/fe-core/src/main/java/org/apache/doris/analysis/AnalyticExpr.java

```diff
@@ -831,6 +831,36 @@ public String toSqlImpl() {
         return sb.toString();
     }
 
+    @Override
+    public String toDigestImpl() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(fnCall.toDigest()).append(" OVER (");
```

Comment:
I think we don't need those `OVER` or `PARTITION BY` keywords in digest?
maybe just a prefix like `AnalyticExpr` is enough.

Reply:
And I'm not quite sure about the performance impact of this implementation.
Looks like these are too much string operations.
Have you test its performance? Especially in high concurrency senario.

And I found some reference about how to implement digest or fingerprint of a SQL:
https://segmentfault.com/a/1190000040132966

Reply:
Thx. 
I've tested locally, there is a 10% query performance loss. It looks bad.

So decide to put it in `auditAfterExec()` and only for slow queries.